packages/store/mud.config.mts

```diff
@@ -1,7 +1,7 @@
 import { storeConfig } from "@latticexyz/cli";
 
 export default storeConfig({
-  storeImportPath: "../",
+  storeImportPath: "../../",
```

Comment:
is this because these things now live in `codegen`?

Reply:
Yep

Reply:
yeah that path is used for default local resolution. It pretty much exists purely to support `tablegen` within `store` package itself

Reply:
added a new issue around this here https://github.com/latticexyz/mud/issues/596